e2e/tests/integration/arrivals-filter.ts

```diff
@@ -34,4 +34,93 @@ describe('Arrivals page filter', () => {
       .get('.danger > :nth-child(1)').contains("TS0123")
 
   });
+
+ //valid service types for filter ("J", "S", "Q", "G", "B", "R", "C", "L")
```

Comment:
rather than a comment, could we just have two variables for use in this test, one called `passengerServiceTypeCode` and one called `nonPassengerServiceTypeCode`